modules/tinymce/src/models/dom/test/ts/browser/table/command/TableDeleteRowTest.ts

```diff
@@ -80,4 +80,43 @@ describe('browser.tinymce.models.dom.table.command.TableDeleteRowTest', () => {
     editor.execCommand('mceTableDeleteRow');
     TinyAssertions.assertContent(editor, initalContent);
   });
+
+  /** Create `rows` number of `tr` elements, with `cols` number of `td` elements inside each.  */
+  const tr = (rows: number, cols: number): string[] =>
+    Arr.range(rows, (r) => '<tr>' + Arr.range(cols, (c) => `<td>${r}-${c}</td>`).join('') + '</tr>');
+
+  Arr.each([
+    {
+      label: 'no colgroup',
+      pathToBody: [ 0, 0 ],
+      createTable: (tbody: string) => `<table><tbody>${tbody}</tbody></table>`,
+    },
+    {
+      label: 'with colgroup',
+      pathToBody: [ 0, 1 ],
+      createTable: (tbody: string) => `<table><colgroup><col><col><col></colgroup><tbody>${tbody}</tbody></table>`,
```

Comment:
Just have to be mindful that the number of `col` elements is fixed here and that could be incorrect from changing `const originalTBody = tr(4, 3);`

Reply:
Good point, I've now refactored it to account for this